assets/js/state/sagas/hosts.js

```diff
@@ -64,6 +73,34 @@ export function* deregisterHost({ payload }) {
   }
 }
 
+export function* checksSelected({ payload }) {
+  const { targetID, targetName } = payload;
```

Comment:
why `targetID` in a host related saga, and not `hostID`?
The same for the `targetName`

Reply:
because the idea is to extract this as `checksSelection` saga, where givent a targetType, targetName and targetID we can orchestrate checks selection feature.

If it helps I can make this totally host specific for now.